.github/workflows/tests.yml

```diff
@@ -94,6 +94,81 @@ jobs:
         run: |
           make osempass
 
+  build-semgrep-js-ocaml-test:
+    name: build semgrep js ocaml for tests
+    runs-on: ubuntu-latest-16-core
+    container: returntocorp/ocaml:alpine-2023-06-16
+    # We need this hack because GHA tampers with the HOME in container
+    # and this does not play well with 'opam' installed in /root
+    env:
+      HOME: /root
+    steps:
+      - name: Make checkout speedy
+        run: git config --global fetch.parallel 50
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      #TODO: we should just call 'make install-deps-for-semgrep-core'
```

Comment:
I originally did that but had some issue (that I totally forget). We should give it another try in a separate PR.